src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -15,14 +15,15 @@
  */
 
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import faker from 'faker';
 
 import Panel, { PanelProps } from './Panel';
 
-const panelContent = <p>{faker.lorem.words()}</p>;
+const panelContent = faker.lorem.words();
 
-const panelHeader = <p>{faker.lorem.words()}</p>;
+const panelHeader = faker.lorem.words();
```

Comment:
Didn't we want to start using strings to make testing easier instead of random words? 😉

Reply:
🤷‍♂️ 

✔️